high_card.rb

```diff
@@ -1,5 +1,12 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+
+ranks = %i["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
```

Comment:
`%i` generates an array of symbols, and is usually followed by `( )`. Not sure if that's what you were trying to do here.

If this is just supposed to be an array, then you don't need anything other than `[ ]`.

There is the `%w` method of creating arrays if you are trying to create an array of strings, but I wouldn't worry about that.